tests/test_permission_to_partial_update_is_supported.py

```
@@ -0,0 +1,188 @@
+        content_type="application/json",
+        data={
+            "books": [
+                book.id  # Referencing related resource created by the user themselves
+            ],
+        },
+    )
+
+    assert response.status_code == status.HTTP_200_OK, response.json()
+    author.refresh_from_db()
+    assert author.books.count() == 1
+    assert (b := author.books.first()) is not None and b.id == book.id
```

Ruff (0.8.2)

Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)